go.sum

```diff
@@ -146,10 +146,12 @@ github.com/cockroachdb/apd v1.1.0/go.mod h1:8Sl8LxpKi29FqWXR16WEFZRNSz3SoPzUzeMe
 github.com/cockroachdb/cockroach-go v0.0.0-20181001143604-e0a95dfd547c/go.mod h1:XGLbWH/ujMcbPbhZq52Nv6UrCghb1yGn//133kEsvDk=
 github.com/cockroachdb/datadriven v0.0.0-20190809214429-80d97fb3cbaa/go.mod h1:zn76sxSg3SzpJ0PPJaLDCu+Bu0Lg3sKTORVIj19EIF8=
 github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd/go.mod h1:sE/e/2PUdi/liOCUjSTXgM1o87ZssimdTWN964YiIeI=
-github.com/codeready-toolchain/api v0.0.0-20210203025716-45f19e752101 h1:oKHlw3jQXcMX8AbAlCdzn8ZS58jnGtjT7ItG5oaistM=
-github.com/codeready-toolchain/api v0.0.0-20210203025716-45f19e752101/go.mod h1:tVouC34ewr4KABqqsVgsUmrg+eDeKsPIBD7VLOeZPEQ=
-github.com/codeready-toolchain/toolchain-common v0.0.0-20210302144622-6e274ed57e69 h1:OSBZqgJ8hOJ29+qh9Qy2qsRRQX2qr7hapu4pIUGCJig=
-github.com/codeready-toolchain/toolchain-common v0.0.0-20210302144622-6e274ed57e69/go.mod h1:2MIk76M6c61igSzapDQr0iyNTB24jOsuCyt7MjnjeZc=
+github.com/codeready-toolchain/api v0.0.0-20210413031854-81652586fd90 h1:CN5cysQBZJl44K+dlFTzWsJxKqUDDco1li8ZS7QpNao=
+github.com/codeready-toolchain/api v0.0.0-20210413031854-81652586fd90/go.mod h1:tVouC34ewr4KABqqsVgsUmrg+eDeKsPIBD7VLOeZPEQ=
+github.com/codeready-toolchain/api v0.0.0-20210512064851-b871cbf562fd h1:f4Fan/pVzKab7arIkPJVuboAel5+Vqveq+qi++cLSuM=
+github.com/codeready-toolchain/api v0.0.0-20210512064851-b871cbf562fd/go.mod h1:tVouC34ewr4KABqqsVgsUmrg+eDeKsPIBD7VLOeZPEQ=
```

Comment:
I'm aware of these two different versions - that's why I created: https://github.com/codeready-toolchain/toolchain-common/pull/172